src/dlgConnectionProfiles.cpp

```diff
@@ -1903,6 +1930,11 @@ bool dlgConnectionProfiles::validateProfile()
         } else {
             notificationArea->show();
             notificationAreaMessageBox->show();
+            if (offline_button) {
+                offline_button->setDisabled(true);
+                offline_button->setToolTip(
+                        QStringLiteral("<html><head/><body><p>%1</p></body></html>").arg(tr("Please set a valid profile name, game server address and the game port before connecting.")));
```

Comment:
:bulb: How about:
```C++
    offline_button->setToolTip(tr("<p>Although these will not be used immediately, please set a valid profile name, game server address and the game port so that the <tt>Reconnect</tt> action can work after the profile has been loaded.</p>"));
```
```

Reply:
No, please don't overcomplicate this so much.
`Please set a valid profile name, game server address and the game port before continuing.`
`Please set a valid profile name, game server address and the game port before opening.`
etc.